sql/moz-fx-data-shared-prod/braze_derived/fxa_win10_users_v1/metadata.yaml

```
@@ -0,0 +1,12 @@
+friendly_name: Fxa Win10 Users
```

```suggestion
friendly_name: Mozilla Account Win10 Users
```
Thinking since we switched from using Fx Account to Mozilla Account we might want to rename this...but it's up to you

---

yeah I think I want to leave this as is for right now. I'm open to that change, but I also think I am trying to keep things as consistent as possible with how this project is being talked about which has currently been all in terms of Fx Accounts....